code/modules/client/loadout/loadout_uniform.dm

```diff
@@ -256,6 +256,10 @@
 	path = /obj/item/clothing/under/costume/swagoutfit
 	cost = 69000
 
+/datum/gear/uniform/misc/janimaid
```

Comment:
add a donkstation comment or move to the /donkstation/ folder as per #72

Reply:
Is that a good enough distrinction?